src/transformers/models/auto/configuration_auto.py

```diff
@@ -28,7 +30,10 @@
 logger = logging.get_logger(__name__)
 
 
-CONFIG_MAPPING_NAMES = OrderedDict(
+_F = TypeVar("_F", bound=Callable[..., Any])
```

Comment:
I used "F" for "Function", could rename this to `_CallableT` (aka Callable Typevar or callable Generic param) to be less obscure.

```suggestion
_CallableT = TypeVar("_CallableT", bound=Callable[..., Any])
```